tests/meta/store/snapshots/snap_test_cg_meta_store_base.py

```diff
@@ -27,7 +28,7 @@
     {
         "archive": False,
         "path": "/path/to/rare-disease/cases/case_id/analysis/sample_id/arriba_ar/sample_id_lanes_1234_trim_arriba.pdf",
-        "tags": ["arriba-ar", "arriba-report", "meta", "mip-rna", "sample_id"],
+        "tags": ["arriba-ar", "arriba_report", "meta", "mip-rna", "sample_id"],
```

Comment:
Snapshottest picked up that the tag is `arriba_report` while it should be `arriba-report`. Should the snapshot  be updated?

Reply:
sharpparrot bundle created a while ago:
![image](https://user-images.githubusercontent.com/26384108/85017691-357a1900-b16c-11ea-8156-3d9272fd2ab5.png)

sharpparrot bundle cerated with this branch just now:
![image](https://user-images.githubusercontent.com/26384108/85017737-4a56ac80-b16c-11ea-870e-42afb17cb83e.png)

Something seems to have changed.

Reply:
strange it looks like this in this branch:
```
  ("arriba_ar", "arriba_report"): {"tags": ["arriba-ar", "arriba-report"], "is_mandatory": True},
```

Reply:
I will checkout it out again on stage and try to recreate the bundle again...